src/main/scala/nl.knaw.dans.easy.deposit/docs/DDM.scala

```diff
@@ -50,19 +52,19 @@ object DDM extends SchemedXml with DebugEnhancedLogging {
         { dm.creators.getNonEmpty.map(author => <dcx-dai:creatorDetails>{ details(author, lang) }</dcx-dai:creatorDetails>) }
         { dm.datesCreated.toSeq.flatMap(_.value).map(str => <ddm:created>{ str }</ddm:created>) }
         { dm.datesAvailable.toSeq.flatMap(_.value).map(str => <ddm:available>{ str }</ddm:available>) }
-        { dm.audiences.getNonEmpty.map(src => <ddm:audience>{ src.key }</ddm:audience>) }
+        { dm.audiences.getNonEmpty.map(src => <ddm:audience>{ src.key.getOrElse("") }</ddm:audience>) }
```

Comment:
Do I understand this code correctly in the sense that you end up with `<ddm:audience/>` entries in your XML when no `src.key` is provided? Would it make sense to clean those?

```suggestion
        { dm.audiences.getNonEmpty.flatMap(_.key).map(src => <ddm:audience>{ src }</ddm:audience>) }
```

Similar for the ones below.

Reply:
The ones below are different.

Reply:
Mostly agree, except for:

* https://github.com/DANS-KNAW/easy-deposit-api/pull/121/files#diff-d235316605173864efd6324ca714a678R136
* https://github.com/DANS-KNAW/easy-deposit-api/pull/121/files#diff-d235316605173864efd6324ca714a678R143